system-status.sh

```diff
@@ -57,4 +57,4 @@ log kubectl -n rook-ceph describe CephFilesystem
 # run "ceph -s" in the toolbox
 log kubectl -n rook-ceph exec \
     "$(kubectl -n rook-ceph get pod -l app=rook-ceph-tools -o jsonpath='{.items[0].metadata.name}')" \
-    ceph -s
+    -- ceph -s
```

Comment:
the separation by `--` is the old notation, and will not be supported in some future versions of `kubectl`. There should be an other/newer way of passing arguments to a command...

Reply:
i don't see anything like that @nixpanic can you provide some link about it? for me, with kube 1.21 it's working fine.

Reply:
Ah, sorry, it is the other way around, usage without `--` is deprecated ([logs](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/k8s-e2e-external-storage-1.21/detail/k8s-e2e-external-storage-1.21/12/pipeline/95)):
```
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
```